docs/installation-docker.rst

```diff
@@ -78,21 +99,169 @@ minutes depending on the size of the data dump.
 This command can be adjusted to restore from an uncompressed database, or
 directly from a ``mysqldump`` command.
 
+Compile locales
+---------------
+Localized string databases are included in their source form, and need to be
+compiled to their binary form::
+
+    docker exec -i kuma_web_1 make localecompile
+
+Dozens of lines of warnings will be printed::
+
+    cd locale; ./compile-mo.sh . ; cd --
+    ./af/LC_MESSAGES/django.po:2: warning: header field 'PO-Revision-Date' still has the initial default value
+    ./af/LC_MESSAGES/django.po:2: warning: header field 'Last-Translator' still has the initial default value
+    ...
+    ./zu/LC_MESSAGES/promote-mdn.po:4: warning: header field 'PO-Revision-Date' still has the initial default value
+    ./zu/LC_MESSAGES/promote-mdn.po:4: warning: header field 'Last-Translator' still has the initial default value
+
+Warnings are OK, and will be fixed as translators update the strings on
+Pontoon_.  If there is an error, the output will end with the error, such as::
+
+    ./az/LC_MESSAGES/django.po:263: 'msgid' and 'msgstr' entries do not both end with '\n'
+    msgfmt: found 1 fatal error
+
+These need to be fixed by a Kuma developer. Notify then in the #mdndev IRC
+channel or open a bug. You can continue with installation, but non-English
+locales will not be localized.
+
+.. _Pontoon: https://pontoon.mozilla.org/projects/mdn/
+
+Generate static assets
+----------------------
+Static assets such as CSS and JS are included in source form, and need to be
+compiled to their final form::
+
+    docker exec -i kuma_web_1 make build-static
+
+A few thousand lines will be printed, like::
+
+    ## Compiling Stylus files to CSS ##
+    compiled build/assets/css/dashboards.css
+    generated build/assets/css/dashboards.css.map
+    ...
+    Post-processed 'css/zones.css' as 'css/zones.718d56a0cdc0.css'
+    Post-processed 'css/zones.css.map' as 'css/zones.css.6be0969a4847.map'
+
+    1717 static files copied to '/app/static', 1799 post-processed.
+
 Visit the Homepage
 ==================
 Open the homepage at http://localhost:8000 . You've installed Kuma!
 
-Coming Soon
-===========
-Docker support is experimental, so the full features of the Vagrant install are
-not supported.  The next steps are enabling and documenting how to:
+Create an admin user
+====================
+Many Kuma settings require access to the Django admin, including
+configuring social login.  It is useful to create an admin account with
+password access for local development.
+
+If you want to create a new admin account, use ``createsuperuser``::
+
+    docker exec -it kuma_web_1 ./manage.py createsuperuser
+
+This will prompt you for a username, email address (a fake address like
+``admin@example.com`` will work), and a password.
+
+If your database has an existing account that you want to use, use the Django
+shell, similar to this::
+
+    docker exec -it kuma_web_1 ./manage.py shell_plus
```

Comment:
I added a few make commands in a separate branch for convenience: https://github.com/jgmize/kuma/commit/69f114c5fdef234799ce748b59565e9e51a3307e, but I wasn't sure of their value vs documenting the docker commands, and neither was @jwhitlock when we discussed it briefly yesterday, so I haven't opened a PR, but I could be easily convinced to open one for further discussion.